.github/wiki/_Sidebar.md

```diff
@@ -1,6 +1,7 @@
 ## Table of Contents
 
 - [[Home]]
+- [[Test the apply action on development]]
```

Comment:
How about `[[Test The Apply Action On Development]]`

Reply:
Fixed in ddc985a